src/calculateRentalCost.js

```diff
@@ -28,7 +28,26 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalCost = 0;
+  const DAILY_RENT = 40;
+  const MEDIUM_TERM = 3;
+  const REG_DISCOUNT = 20;
+  const LONG_TERM = 7;
+  const BIG_DISCOUNT = 50;
+
+  for (let i = 0; i < days; i++) {
+    totalCost += DAILY_RENT;
+  }
+
+  if (days >= MEDIUM_TERM && days < LONG_TERM) {
+    totalCost -= REG_DISCOUNT;
+  }
+
+  if (days >= LONG_TERM) {
+    totalCost -= BIG_DISCOUNT;
+  }
```

Comment:
4. [CHECKLIST]Prefer `if` with `return` over `if else` to simplify later conditions.

If you use `return` inside an `if`, you will not need to write double conditions